app/models/record.js

```diff
@@ -72,10 +72,30 @@ const Record = Model.extend(Validations, Copyable, {
   }),
   json: attr('json', {
     defaultValue() {
+      // Use a static fallback version when service is not available during initialization
+      let schemaVersion = '2.13.0'; // fallback version
```

Comment:
@dvonanderson newly created records are assigned the fallback version, not the actual schema version